launchers/bin/msu.py

```diff
@@ -65,16 +65,20 @@ def parseargs(argv=None):
     required.add_argument('-e', '--email', required=True,
         help=('The email address of the login.ubuntu.com account to be created on the device.')
         )
-    parser.add_argument('-p', '--password', 
+    parser.add_argument('-p', '--password',
         help=('The password of the account to be created on the device. This password is not saved. Either this or --ssh-keys is required.')
         )
-    parser.add_argument('--until', 
+    parser.add_argument('--since-days-ago',
+        default='2',
+        help=('Optionally specify how many days ago the since field will use, default 2 days.')
```

Comment:
Can you please use the following help text: 'Optionally specify how many days ago the "since" field will use: default is 2 days. The "since" and the "until" fields define the valid period of the system's time: if the system time is not in the valid period, the user is not created from the assertion.'

Reply:
Done!